scripts/install_dependencies/install_openvino_dependencies.sh

```diff
@@ -148,6 +148,7 @@ elif [ "$os" == "rhel8" ] ; then
 
     pkgs_core=(libtbb2 libpugixml1v5)
     pkgs_opencv_req=(gtk3)
+    pkgs_python=(python38 python38-devel python38-setuptools python38-pip)
```

Comment:
1. why do we need setuptools? it should be already a dependency of python38-pip
2. instead of python38-devel we need something like libpython3.8 (don't know exact name, it's a name from U20.04)